root/scripts/talker/coach.txt

```diff
@@ -5387,7 +5397,7 @@ Rule SurvivorSpottedMudmenCoach
 
 Response SurvivorSpottedMudmenC3M2Coach
 {
-	scene "scenes/Coach/SeeMudmen01.vcd"  //Mudmen!
+	scene "scenes/Coach/GooedBySpitterC103.vcd"  //You have got to be kidding me.
```

Comment:
Generic line that shouldn't even be used for mudmen. The only exception currently seems to be finding a specific pill spawn on Swamp Fever, I'll just cut that scene from that rule instead.